src/plugins/irc/irc-server.c

```diff
@@ -2098,34 +2097,26 @@ irc_server_send_one_msg (struct t_irc_server *server, int flags,
         ptr_msg = (new_msg) ? new_msg : message;
 
         msg_encoded = NULL;
-        irc_message_parse (server, ptr_msg, NULL, NULL, NULL, NULL, NULL, NULL,
-                           NULL, NULL, NULL, NULL, &pos_channel, &pos_text);
-        if (weechat_config_boolean (irc_config_network_channel_encode))
-            pos_encode = (pos_channel >= 0) ? pos_channel : pos_text;
+        ptr_chan_nick = (channel) ? channel : nick;
+        if (ptr_chan_nick)
+        {
+            snprintf (modifier_data, sizeof (modifier_data),
+                      "%s.%s.%s",
+                      weechat_plugin->name,
+                      server->name,
+                      ptr_chan_nick);
+        }
```

Comment:
The then part of this if should be eliminated to be consistent with the decode part (not using nick/channel part in modifier)